### DISCUSSION TITLE
Rover Support

### DISCUSSION THREAD

**Discussion by goofy78270** [Kong/insomnia#3852] [Ideas]
I was looking to see if insomnia would be open, or has plans, to support rover and graphQl? Our company recently added rover support to show contact info per service, but it is only available within Apollo itself, and not through an introspection query.

reference link discussing change:
https://www.apollographql.com/docs/studio/federated-graphs/#subgraph-contact-info

**Comment by dimitropoulos** [comment]
I'm not familiar with rover, but it certainly sounds interesting.

Could you please say a little more about what supporting rover would look like or what new functionality it would ultimately result in?

**Comment by goofy78270** [comment]
Rover is simply a different CLI to publish graph schemas - https://www.apollographql.com/docs/studio/schema/cli-registration/#registering-a-non-federated-schema

With Rover, comes different capabilities, such as the contact info mentioned above, but it also offers better management of federated services withing graph.
